frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -195,7 +228,19 @@ const InspectionRow = ({ inspection, openDialog, setMissions, openScheduledDialo
     )
 }
 
-const columns = ['Status', 'Name', 'Description', 'Area', 'Last completed', 'Deadline', 'Add to queue']
+const SmallScreenInfoText = () => {
+    const { TranslateText } = useLanguageContext()
+    return (
+        <div id="SmallScreenInfoText">
+            <Icon name={Icons.Info} size={24}></Icon>
+            <Typography>
+                {TranslateText(
+                    'Some columns in the table are hidden due to small screen. Rotate the screen to show more.'
```

Comment:
I would name this Small Screen Info Text and then have the hole sentence in the .json files

Reply:
https://github.com/equinor/flotilla/pull/1338
Have done that in this one so will fix it there if this one is approved first